src/fun/counting/stats.go.tmpl

```diff
@@ -0,0 +1,73 @@
+{{/*
+        Counting statistics
+        
+        Author: H1nr1 <https://github.com/H1nr1>
+*/}}
+
+{{/* Configurable Values */}}
+{{ $LBLength := 10 }} {{/* How many members to show on leaderboard; MAX OF 100 */}}
+{{/* End of configuration values */}}
+
+{{/* No Touchy */}}
+{{/* Initializing variables */}}
+{{ $Current := sub (dbGet 0 "Next").Value 1 }}
+{{ $CCount := (dbGet .User.ID "CCount").Value }}{{ $CCorrect := (dbGet .User.ID "CCorrect").Value }}
+{{ $UserID := 0 }}
+{{ $SecondChance := "" }}{{ if eq (toInt (dbGet 0 "CSecondChance").Value) 0 }}{{ $SecondChance = "1" }}{{ else if ge (toInt (dbGet 0 "CSecondChance").Value) 1 }}{{ $SecondChance = "0" }}{{ end }}
+{{ $Args := parseArgs 0 "" (carg "string" "Who") }}
+
+{{ if not ($Args.IsSet 0) }} {{/* Server Stats */}}
+	{{ sendMessage nil (cembed 
+		"title" "**🔢 Counting Statistics**"
+		"description" (print "⌚ __Current Score:__ " $Current "\n🏅 __High Score:__ " (dbGet 0 "CHighScore").Value " on " (formatTime (dbGet 0 "CHighScore").UpdatedAt "01/02") " by " (userArg (toInt (dbGet 0 "CHSUser").Value)) "\n⏮️ __Last Counter:__ " userArg (dbGet 0 "LastUser").Value "\n💾 __Saves Remaining:__ " $SecondChance)
+		"color" 30654
+		"timestamp" currentTime) }}
+	{{ deleteTrigger 3 }}
+	{{ return }}
+
+{{ else if (inFold (cslice "Me" "My" "0") ($Args.Get 0)) }} {{/* Triggering user's stats */}}
+	{{ sendMessage nil (cembed
+		"title" (print "**🔢 " .User.Username "'s Counting Statistics**")
+		"description" (print "<@" .User.ID "> has counted a __total__ of **" $CCount " times**\n<@" .User.ID "> has counted __correctly__ **" $CCorrect " times**\nThis makes <@" .User.ID ">'s __average__ **" (div (round (mult (div $CCorrect $CCount) 10000)) 100) "%**")
+		"color" 30654
+		"timestamp" currentTime) }}
+
+{{ else if and (reFind `\d{17,19}` ($Args.Get 0)) }} {{/* User mentioned */}}
+	{{ if reFind `<@\d{17,19}>` ($Args.Get 0) }} {{/* if mention */}}
+		{{ $UserID = slice ($Args.Get 0) 2 (sub (len ($Args.Get 0)) 1) }}
+	{{ else if reFind `\d{17,19}` ($Args.Get 0) }} {{/* if user ID */}}
+		{{ $UserID = $Args.Get 0 }}{{ end }}
```

Comment:
This can be condensed, there is absolutely no need for this branching.

You are already extracting the mentioned user's ID in line 35 with your `reFind`, use that.